frame/support/test/tests/benchmark_ui/bad_param_range.stderr

```diff
@@ -1,5 +1,7 @@
-error: The start of a `ParamRange` must be less than or equal to the end
-  --> tests/benchmark_ui/bad_param_range.rs:10:21
-   |
-10 |     fn bench(x: Linear<3, 1>) {
-   |                        ^
+error[E0080]: evaluation of constant value failed
+ --> tests/benchmark_ui/bad_param_range.rs:5:1
+  |
+5 | #[benchmarks]
+  | ^^^^^^^^^^^^^ attempt to compute `0_usize - 1_usize`, which would overflow
```

Comment:
Yea not sure how helpful this error is. Maybe we just do it at runtime?

Reply:
Added a runtime check.